### PR TITLE
HSEARCH-4220 Remove horizontal scrolling from the reference documentation

### PR DESCRIPTION
**Pull request by yrodiere**
https://hibernate.atlassian.net/browse/HSEARCH-4220

### CODE REVIEW

**Comment by yrodiere** [comment]
Jenkins, retest this please

File: documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc [hunk 1]
```diff
@@ -66,7 +66,7 @@ Value fields as well as <<mapper-orm-bridge-index-field-dsl-object,object fields
 |`@PropertyBinding`
 |`@TypeBinding`
 |<<mapper-orm-identifiermapping,`@DocumentId`>>
-|`@Indexed(routingBinder = ...)`
+|`@Indexed( routingBinder = ... )`
```
Comment (fax4ever) [inline documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc:69]: Shall we add a space after the parenthesis?

**Review by fax4ever — APPROVED** [review]
Thanks! Just one doubt!

File: documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc (hunk 1 above)
Comment (yrodiere) [reply documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc:69]: After the closing one? I don't see why we would.

I added these spaces to reduce a bit the width of the cell, that's all.

**Comment by yrodiere** [comment]
Merged, thanks

File: documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc (hunk 1 above)
Comment (fax4ever) [reply documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc:69]: I meant after the opening one, anyway it's OK

Comment (yrodiere) [reply documentation/src/main/asciidoc/reference/mapper-orm-bridge-basics.asciidoc:69]: Then that's exactly what I did, and I don't understand the question :) Maybe I already answered.

**Pull request merged**